1.MLP_Binary_Classifier/MLP_utils/utils.py

```diff
@@ -0,0 +1,627 @@
+#!/bin/env/python
```

Comment:
Please add a docstring at the top of the file (see a previous comment)

Also, is the python shebang necessary?